example/ios/Podfile

```diff
@@ -25,7 +25,7 @@ if linkage != nil
   use_frameworks! :linkage => linkage.to_sym
 end
 
-target 'ExampleApp' do
+target 'exampleApp' do
```

Comment:
We could remove flipper. It was fixed specifically for 73. That fix was not backported to 71 or 72. And it is removed entirely in 74.